docs/ovn_k8s.conf.5

```diff
@@ -23,45 +22,66 @@ This represents the conntrack zone used for the conntrack flow rules.
 .PP
 .SH [Logging]
 .TP
-\fBlogfile\fR=/var/log/openvswitch/ovn-k8s-cni-overlay.log
-Logfile name (with path) for ovnkube to write to.
+\fBlogfile\fR=
+When logging to a file, this is the path and file. E.g., /var/log/openvswitch/ovn-k8s-cni-overlay.log
+Default is not to log to a file.
 .TP
-\fBloglevel\fR=0
-Level is the logging verbosity level: 5=debug, 4=info, 3=warn, 2=error, 1=fatal (default: 0).
+\fBloglevel\fR=4
+Level is the logging verbosity level: 5=debug, 4=info, 3=warn, 2=error, 1=fatal (default: 4).
 .SH [CNI]
 .PP
 The following config values are used for the CNI plugin.
 .TP
-\fBcni-conf-dir\fR=/etc/cni/net.d
+\fBconf-dir\fR=/etc/cni/net.d
 Directory that contains the cni plugin.
```

Comment:
This should also be updated in my opinion, conf-dir it's the CNI config directory in which ovnkube writes the overlay CNI config file. Details: https://github.com/openvswitch/ovn-kubernetes/blob/master/go-controller/pkg/config/config.go#L70-L71

Reply:
Yes. This should convey that the location is for CNI configuration file.
Perhaps this?
'Directory that contains the cni plugin configuration.'

Reply:
I'm not sure if it's confusing to who's reading it, since it's the directory where the CNI plugin configuration will be written.